src/pages/Home/index.tsx

```diff
@@ -0,0 +1,15 @@
+import HotNews from "../../widgets/HotNews";
+import NewsFeed from "../../widgets/NewsFeed";
+
+function HomePage() {
+  document.title = 'News Portal: Main page';
```

Comment:
It will be run each render
Look at this https://dev.to/luispa/how-to-add-a-dynamic-title-on-your-react-app-1l7k

Reply:
done too